keter.cabal

```diff
@@ -108,11 +108,11 @@ Library
                        Data.Conduit.Process.Unix
   ghc-options:         -Wall
   c-sources:           cbits/process-tracker.c
-
+  hs-source-dirs: src
 Executable keter
   default-language:    Haskell98
   Main-is:             keter.hs
-  hs-source-dirs:      main
+  hs-source-dirs:      src/main
```

Comment:
normally we put the executables in a folder called `app` instead of `src`, but this also works fine.